packages/app/src/cli/models/extensions/specifications/app_config_webhook_schemas/webhook_subscription_schema.ts

```diff
@@ -0,0 +1,28 @@
+import {UriValidation, removeTrailingSlash} from '../validation/common.js'
+import {zod} from '@shopify/cli-kit/node/schema'
+
+export enum ComplianceTopic {
+  CustomersRedact = 'customers/redact',
+  CustomersDataRequest = 'customers/data_request',
+  ShopRedact = 'shop/redact',
+}
+
+export const WebhookSubscriptionSchema = zod.object({
+  topics: zod
+    .array(zod.string({invalid_type_error: 'Values within array must be a string'}), {
+      invalid_type_error: 'Value must be string[]',
+    })
+    .optional(),
+  uri: zod.preprocess(removeTrailingSlash, UriValidation, {required_error: 'Missing value at'}),
+  sub_topic: zod.string({invalid_type_error: 'Value must be a string'}).optional(),
+  include_fields: zod.array(zod.string({invalid_type_error: 'Value must be a string'})).optional(),
```

Comment:
Is `zod` default messaging not clear enough when the type is wrong?

Reply:
This was an existing schema that I moved out of the `webhooks` specification file to its own individual file, since it's shared across 3 specifications (`webhooks`, `privacy_compliance`, and now `webhook_subscription`). So I don't have context on how this was implemented 😬